### PR TITLE
Add unit tests for S3 proxy view.

### PR DESCRIPTION
**Pull request by jmcarp**
EOM

### CODE REVIEW

File: regulations/settings/base.py [hunk 1]
```diff
@@ -229,4 +229,7 @@
     'regulations.generator.sidebar.help.Help',
 )
 
+BUCKET = os.getenv('BUCKET')
+ACCESS_KEY_ID = os.getenv('ACCESS_KEY_ID')
+SECRET_ACCESS_KEY = os.getenv('SECRET_ACCESS_KEY')
```
Comment (cmc333333) [inline regulations/settings/base.py]: Should have pointed this out before, but it'd be helpful to namespace these a little better. You could imagine there's an S3 bucket dedicated to static assets, for example.

File: regulations/tests/views_comment_tests.py [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+        get_random.return_value = 'not-so-random'
+        resp = self.client.get('/comments/attachment?size=42')
+        self.assertEqual(resp.status_code, 200)
+        body = json.loads(resp.content.decode())
+        generate_presigned.assert_called_with(
+            ClientMethod='put_object',
+            Params={
+                'ContentLength': 42,
+                'ContentType': 'application/octet-stream',
+                'Bucket': settings.BUCKET,
```
Comment (cmc333333) [inline regulations/tests/views_comment_tests.py]: it'd be a tad clearer to use the constant defined above, 'test-bucket', the same way you're using 42.

**Comment by cmc333333** [comment]
Just one little thing. I'll make a separate issue for renaming the settings unless you want to tackle that here.

File: regulations/settings/base.py (hunk 1 above)
Comment (jmcarp) [inline regulations/settings/base.py]: Agreed, will make the change now.

**Comment by cmc333333** [comment]
:+1:

**Pull request merged**